.github/workflows/testing.yml

```diff
@@ -210,3 +210,17 @@ jobs:
         run: |
           cd semantic-search-example
           npm run test
+
+  end-to-end:
+    name: end-to-end
+    runs-on: ubuntu-latest
+    if: always() # Run teardown even if the tests fail
```

Comment:
nit: This isn't a teardown job though, right?

Reply:
Ah must've been a c&p error, apologies! I'll remove.

Reply:
Ah, no, leaving this here, actually -- the e2e tests include a teardown job, so even if the test fails, the created index will be torn down, ensuring we don't just create orphan indexes in the event that these tests fail.